clickhouse_driver/connection.py

```diff
@@ -122,7 +127,7 @@ class Connection(object):
     """
 
     def __init__(
-            self, host, port=None,
+            self, host, port=None, server_hostname=None,
```

Comment:
`server_hostname` parameter should be placed at the end or in the "Secure socket parameters" section right after `keyfile=None, certfile=None` which is preferrable.

This will keep backwards compatibility for those who constructs client like `Client('localhost', 9000, 'user', 'pass', 'mydb')`

Reply:
ah, good feedback, done